examples/lb3-application/lb3app/server/config.json

```diff
@@ -1,11 +1,10 @@
 {
-  "restApiRoot": "/api",
-  "host": "0.0.0.0",
-  "port": 3000,
   "remoting": {
+    "errorHandler": false,
     "context": false,
     "rest": {
       "handleErrors": false,
+      "handleUnknownPaths": false,
```

Comment:
Please consider describing this flag in the page https://loopback.io/doc/en/lb4/migration-mounting-lb3app.html too. IMO, it should be enabled at the time when LB3 app is mounted inside LB4 context, otherwise it's not possible to use LB3 app with LB3 `restApiRoot` set to `/`, regardless of the booter model (fullApp vs restRouter).

Reply:
That example still uses `middleware.json` to load `loopback.rest`, so it's not required there, unless we want to modify the application code there.

I have included the required info in this PR.